app/test_main.py

```diff
@@ -2,2 +1,43 @@
+import pytest
 
-# write your code here
+
+from app import main
+
+def convert_to_human(animal_age: int,
+                     first_year: int,
+                     second_year: int,
+                     each_year: int
+) -> int:
+    if animal_age < first_year:
+        return 0
+    if animal_age < first_year + second_year:
+        return 1
+    return 2 + (animal_age - first_year - second_year) // each_year
+
+
+@pytest.mark.parametrize(
+    "first_year,second_year,each_year_cat,each_year_dog",
+    [
+        (14, 9, 4, 5),
+        (15.1, 9, 4, 5),
+        (15, 8, 4, 5),
+        (15, 9.1, 4, 5),
+        (15, 9, 3, 4),
+        (15, 9, 4.1, 5.1),
```

Comment:
The parameters `each_year_cat` and `each_year_dog` are expected to be integers, but here they are floating-point numbers (4.1 and 5.1). According to the task description, these values should be integers.